easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -87,6 +87,48 @@ def setup_cmake_env(tc):
     setvar("CMAKE_LIBRARY_PATH", library_paths)
 
 
+def setup_cmake_env_python_hints(cmake_version):
+    """Convenience function to set CMake hints for FindPython[_2/3] as environment variables.
+    Needed to avoid wrong Python being picked up by CMake when not called directly by EasyBuild but as step in a
+    build and no option is provided to set custom CMake variables.
```

Comment:
```suggestion
    """Set environment variables as hints for CMake to prefer the Python module, if loaded.
    Useful when there is no way to specify arguments for CMake directly,
    e.g. when CMake is called from within another build system.
    Otherwise get_cmake_python_config_[str/dict] should be used instead.
```